awesomplete.js

```diff
@@ -279,9 +279,19 @@ _.SORT_BYLENGTH = function (a, b) {
 
 function regEscape(s) { return s.replace(/[-\\^$*+?.()|[\]{}]/g, "\\$&"); }
 
-$$("input.awesomplete").forEach(function (input) {
-	new Awesomplete(input);
-});
+function init() {
+	$$("input.awesomplete").forEach(function (input) {
+		new Awesomplete(input);
+	});
+}
+
+// DOM already loaded?
+if (document.readyState !== 'loading') {
```

Comment:
1) using double quotes now

2) state of documentReady goes from `loading` to `interactive` to `complete` (https://developer.mozilla.org/en-US/docs/Web/API/document.readyState). DOMContentLoaded fires with the `loading` -> `interactive` transition, so `loading` is the only state when listening to it makes sense.